bowling/main.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include <string>
+
+#include "game.hpp"
+
+void oneParameter(const std::string& param) {
+    if (param.front() == '-') {
+        if (param == "-h" || param == "--help") {
+            Game::printHelp();

```

Comment:
why it's the game class that prints such help?

Reply:
done - moved to main